src/App/component/AddProjectDialog.jsx

```diff
@@ -15,32 +13,22 @@ import {
 
 export default function AddProjectDialog({open, reloadProjects, handleClose}) {
   const [projectName, setProjectName] = useState("")
-  const [githubRepositoryURL, setGithubRepositoryURL] = useState("")
-  const [sonarRepositoryURL, setSonarRepositoryURL] = useState("")
-  const setIsGithubAvailable = useState(false)[1]
-  const setIsSonarAvailable = useState(false)[1]
   const jwtToken = localStorage.getItem("jwtToken")
 
   const createProject = () => {
     let checker = []
-    if (projectName === "" || (githubRepositoryURL === "" && sonarRepositoryURL === "")) {
+    if (projectName === "") {
       alert("不準啦馬的>///<")
     } else {
-      if (githubRepositoryURL !== "") {
-        checker.push(checkGithubRepositoryURL());
-      }
-      if (sonarRepositoryURL !== "") {
-        checker.push(checkSonarRepositoryURL());
-      }
 
       Promise.all(checker)
         .then((response) => {
           if (response.includes(false) === false) {
             let payload = {
```

Comment:
Could you please help me change this variable to be a constant ?
```suggestion
            const payload = {
```